scripts/blocklist-skylink.sh

```diff
@@ -3,14 +3,49 @@
 set -e # exit on first error
 
 if [ -z "$1" ]; then
-    echo "Please provide a skylink to blocklist" && exit 1
+    echo "Please provide either a skylink or file with skylinks separated by blank lines" && exit 1
 fi
 
+#########################################################
+# read either a file containing skylinks separated by new 
+# lines or a single skylink and put them in an array
+#########################################################
+skylinks=()
+if test -f "$1"; then
+    OLDIFS=$IFS
+    IFS=','
```

Comment:
not following why you adjust `IFS` to be a comma, but never mention comma separation

Reply:
me neither, it just doesn't work without it and all SO examples have it there, it can be empty fwiw and it works too but it has to be set

Reply:
fwiw I found:
```
#!/bin/bash
input="/path/to/txt/file"
while IFS= read -r line
do
  echo "$line"
done < "$input"
```
but don't mind if we have it in there - it's just confusing... perhaps set it to '' at least